cognee/infrastructure/databases/vector/create_vector_engine.py

```
@@ -134,6 +134,28 @@ def create_vector_engine(
             embedding_engine=embedding_engine,
         )
 
+    elif vector_db_provider == "neptune_analytics":
+        try:
+            from langchain_aws import NeptuneAnalyticsGraph
```

Ruff (0.12.2)

`langchain_aws.NeptuneAnalyticsGraph` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)